From 525f70d2011183232f91d10f3e840a33247dad55 Mon Sep 17 00:00:00 2001
From: Randy <rrandymiller@gmail.com>
Date: Mon, 17 Nov 2025 21:58:30 -0500
Subject: [PATCH] modified 2 lines firstfile

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index f00c965..bb076e7 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -6,5 +6,5 @@
 6
 7
 8
-9
-10
+-9
+-10
-- 
2.43.0

